src/Pages/OffersPage.tsx

```diff
@@ -0,0 +1,83 @@
+import { fetchOffers } from '../redux/slices/offersSlice'
```

Comment:
please see comment above this should be a list of offers component. Prop that accepts list of offers and state. Fetching logic is done in parent